src/datagrid/pagination.component.jsx

```diff
@@ -244,6 +244,7 @@ const paginationComponent = (WrappedComponent) => {
                 {intl.formatMessage({ id: 'Grid.Pagination.RowsOnPage' })}
               </RowsOnPageLabel>
               <RowsOnPageSelect
+                id="rowsOnPageSelect"
```

Comment:
Hard coded id's are bad in case there's multiple grids on a page. Could grid ID be used here as a prefix?